doc/guides/getting_started/src/SUMMARY.md

```diff
@@ -8,6 +8,7 @@
 - [Formal Verification](setup_formal.md)
 - [Building (and Testing) Software](build_sw.md)
 - [Building Documentation](build_docs.md)
+- [Running e2e Tests](https://opentitan.org/book/sw/device/silicon_creator/rom/doc/e2e_tests.md)
```

Comment:
It may be better not to have this page in the index of a book it isn't in. You could link to this guide from within `build_sw.md` in the getting started guide, if you want to direct people to it from the getting started guide.